Automattic/jetpack
# Storybook: Use `ThemeProvider` instead of `base-styles`

Following up on the proposal of `ThemeProvider` on top of `base-styles`.

#### Changes proposed in this Pull Request:

* Remove `base-styles` and add the usage of `ThemeProvider`.

#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Start Storybook
* Nothing should change, and all components still receiving CSS Vars.
